CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Added `assert_files_equals`, `assert_files_not_equals`
 - Fixed total tests wrong number
 - Added `BASHUNIT_TESTS_ENV`
+- Added support for running tests in Alpine
```

Comment:
Please, do a git pull to fetch the latest changes as the 0.16 was released this weekend, so you can add this line in a new section on top as `## Unreleased`